### DISCUSSION TITLE
10th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3260] [General]
Hi everyone,

I (@mxinden) will be in the open maintainers call **tomorrow (Tuesday)**. Not sure who else will join given the end of the year approaching. See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

See you all tomorrow.

**Comment by thomaseizinger** [comment]
I'll be there.

**Comment by mxinden** [comment]
Pointers shared during the meeting:

- Max has pager setup, thus no need to be online over the break.
- CI failures due to protoc installation rate limiting are slowing us down. Any quick fix ideas?
    - Just add a token: https://github.com/subspace/subspace/blob/bfda79167fe3fc83d60786b3f55f944a8915e6aa/.github/workflows/rust.yml#L42-L43
    - https://github.com/libp2p/rust-libp2p/pull/3258
- Kademlia backpressure: https://github.com/libp2p/rust-libp2p/discussions/3235
    - quick fix: buffer outbound messages if we have more than 32 in flight at the moment
    - introduce jitter for announcements to prevent them all triggering at the same time
- kademlia: API consistency between PUT record/provider and queries: Should we also remove quorums from PUT and have the user "stop" propagation once enough PUTs have been successful?
- Show off Kademlia exporter metrics
    - kademlia-exporter.max-inden.de/
- Protocol upgrades
    - https://github.com/libp2p/rust-libp2p/issues/2863
    - https://github.com/libp2p/rust-libp2p/issues/3268
- connection management: https://github.com/libp2p/rust-libp2p/pull/3254
- Release automation
- New workspace/dependency structure: https://github.com/libp2p/rust-libp2p/discussions/3072#discussioncomment-4394785
- No cyclic dependencies: https://github.com/libp2p/rust-libp2p/pull/3023
- IPFS Thing
    - https://discuss.ipfs.tech/t/pre-registration-is-open-for-ipfs-thing-april-2023/15707
